esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -4276,43 +4302,65 @@ void CWsWuFileHelper::createZAPWUQueryAssociatedFiles(IConstWorkUnit* cwu, const
     }
 }
 
-void CWsWuFileHelper::setZAPFile(const char* zipFileNameReq, const char* zipFileNamePrefix,
-    StringBuffer& zipFileName, StringBuffer& zipFileNameWithPath)
+void CWsWuFileHelper::setZIPFileName(const char* tempDirName, StringBuffer& zipFileNameWithFullPath)
 {
-    StringBuffer outFileNameReq(zipFileNameReq);
-    //Clean zipFileNameReq. The zipFileNameReq should not end with PATHSEPCHAR.
-    while (!outFileNameReq.isEmpty() && (outFileNameReq.charAt(outFileNameReq.length() - 1) == PATHSEPCHAR))
-        outFileNameReq.setLength(outFileNameReq.length() - 1);
-
-    if (outFileNameReq.isEmpty())
-        zipFileName.set(zipFileNamePrefix).append(".zip");
-    else
-    {
-        zipFileName.set(outFileNameReq.str());
-        const char* ext = pathExtension(zipFileName.str());
-        if (!ext || !strieq(ext, ".zip"))
-            zipFileName.append(".zip");
-    }
+#ifdef _CONTAINERIZED
+    zipFileNameWithFullPath.set("/var/lib/HPCCSystems/");
```

Comment:
shouldn't be hardwired - should also use getConfigurationDirectory,
Should be common code with both using "temp" not "run"

Reply:
The setZIPFileName() will be removed.